### PR TITLE
[MM-19223] Created reduced size notification icon to fix pixelation

### PR DESCRIPTION
**Pull request by devinbinnie**
**Summary**
This PR creates a reduced icon (48px in size) to use with notifications since Windows 10 was not downsizing our large 512px size icon.

**Issue link**
https://mattermost.atlassian.net/browse/MM-19223

### CODE REVIEW

File: src/browser/components/MattermostView.jsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ const preloadJS = `file://${remote.app.getAppPath()}/browser/webview/mattermost_
 const ERR_NOT_IMPLEMENTED = -11;
 const U2F_EXTENSION_URL = 'chrome-extension://kmendfapggjehodndflmmgagdbamhnfd/u2f-comms.html';
 
-const appIconURL = `file:///${remote.app.getAppPath()}/assets/appicon.png`;
```
Comment (Willyfrog) [inline src/browser/components/MattermostView.jsx]: should we downsize all OSs?

Comment (devinbinnie) [reply src/browser/components/MattermostView.jsx]: It seems to only be an issue on Windows, and I think that's the only OS using an image we provide. I'm pretty sure Mac just uses the icon from the application itself.

**Comment by Willyfrog** [comment]
it is, on any other platform should be no change

**Comment by Willyfrog** [comment]
ah! sorry @jgilliam17 i thought you approved, and I just realized you were testing on mac
I'll revert

**Comment by ogi-m** [comment]
Tested on Windows and looks good, the notification icon doesn't appear pixelated

**Comment by jgilliam17** [comment]
Thank you @ogi-m 
Looks good to merge @Willyfrog

**Comment by Willyfrog** [comment]
it was already merged, never got to revert it (created the branch, but never did merge the revert)

**Pull request merged**